README.md

```diff
@@ -459,23 +457,27 @@ The steps to activate your BBG using GWE can be found here:
 
 A summarized version of the steps are included here:
 
-```
+```sh
 $ ssh debian@<IP Address>
-$ sudo gwe --set-product-id <Product ID> --set-uuid <MAC Address>
+$ sudo gwe --set-product-id <Product ID> --set-uuid <MAC Address> --set-update-interval 300
 $ sudo reboot
 ```
 
-## Coding the sensors
+## Installing a GWE application
 
 The BBG needs to be able to talk to the Temp/Humidity sensor.
 
-In a web browser, type in the IP address of the BBG in the following format:
+We will use the OTA features of GWE to install the th02.py script.
 
-```
-http://xxx.xxx.xxx.xxx:3000/
-```
+ This will read data from the temperature and humidity sensor and send the data to your Murano solution.
 
-Copy the code from TH02.py into the code area and click run. This will read data from the temperature and humidity sensor and send the data to your Murano solution.
+```sh
+$ cd gwe.apps/th02
+$ tar zcvf ../th02.v1.tar.gz install.sh supervisor.conf th02.py
+$ cd ..
+$ murano content upload th02.v1.tar.gz th02.v1.tar.gz
+$ murano product device write <mac address> engine_fetch \{\"install\":\[\{\"name\":\"th02.v1.tar.gz\"\}\]
```

Comment:
<mac address> is ... serial number is... we specifically use mac address in this tutorial which is why we use it here? (just validating)